packages/taiko-client/internal/testutils/interfaces.go

```diff
@@ -17,6 +17,12 @@ type ChainSyncer interface {
 type Proposer interface {
 	utils.SubcommandApplication
 	ProposeOp(ctx context.Context) error
-	ProposeTxLists(ctx context.Context, txLists []types.Transactions, parentMetaHash common.Hash, l2BaseFee *big.Int) error
+	ProposeTxLists(
+		ctx context.Context,
+		txLists []types.Transactions,
+		parentMetaHash common.Hash,
+		l2BaseFee *big.Int,
+		isSignalForcePropose bool,
```

Comment:
Nitpick - maybe this is too many parameters already and turning this bool into an enum or having a separate `ForceProposeTxLists` would be clearer.